src/go/k8s/pkg/resources/certmanager/pki.go

```diff
@@ -53,7 +53,7 @@ func NewPki(
 }
 
 func (r *PkiReconciler) prepareRoot(
-	prefix string, san []string,
```

Comment:
Why this is removed? The schema registry with TLS will be not working after this change lands.

Reply:
Why not?

Reply:
NVM I saw that schemaregistry_api.go is correctly setting up the `DNSNames`.